### PR TITLE
Add seed param

### PR DESCRIPTION
**Pull request by bodepd**

### CODE REVIEW

**Comment by bodepd** [comment]
@hkumarmk  this passed an [integration test](https://github.com/JioCloud/puppet-rjil/pull/418) and is ready to be merged.

File: manifests/config.pp [hunk 1]
```diff
@@ -332,29 +334,31 @@
...
 
-  contrail_linklocal {'metadata':
-    ensure                  => present,
-    ipfabric_service_address=> $nova_metadata_address,
-    ipfabric_service_port   => $nova_metadata_port,
-    admin_password          => $keystone_admin_password,
-    service_address         => '169.254.169.254',
-    service_port            => 80,
-    require                 => Service['contrail-api'],
+    contrail_linklocal {'metadata':
```
Comment (hkumarmk) [inline manifests/config.pp]: This is also to be run only on seed node I believe

Comment (bodepd) [inline manifests/config.pp]: it is, I think you misread the diff

Comment (hkumarmk) [inline manifests/config.pp]: oh never mind... I see you doing that.

Comment (hkumarmk) [inline manifests/config.pp]: yeah :)

**Comment by hkumarmk** [comment]
In fact you should add seed param in init.pp also, as contrail::config class is calling from there also.

**Comment by hkumarmk** [comment]
@bodepd Other than above comment, Its good to go! :+1:

**Comment by hkumarmk** [comment]
Merging this as it passed the test (#700)

**Pull request merged**